undead-discord.css

```diff
@@ -79,7 +79,7 @@
 /* .status needs to be before .status-typing */
 
 .status {
-  border-radius: 33% !important;
+  border-radius: 0px !important;
```

Comment:
Because of the global `* { border-radius: unset !important }` this block can be removed.
For a consistent experience the `.status-typing` block should also be deleted.